autopush/websocket.py

```diff
@@ -219,6 +241,11 @@ def process_notifications(self, tries=0):
         if self._should_stop:
             return
 
+        # Are we paused? Try again later.
+        if self.paused:
+            reactor.callLater(1, self.process_notifications)
```

Comment:
Could we call `process_notifications` in `resumeProducing`? (Setting short-lived timers for polling variables like this gets expensive in JavaScript, but I've seen this used elsewhere in the code. I assume this is the Pythonic way to do it?)